ipaserver/install/krainstance.py

```diff
@@ -274,13 +273,7 @@ def __add_vault_container(self):
             'vault.ldif', {'SUFFIX': self.suffix}, raise_on_err=True)
 
     def __apply_updates(self):
-        sub_dict = {
-            'SUFFIX': self.suffix,
-        }
-
-        ld = ldapupdate.LDAPUpdate(dm_password=self.dm_password,
-                                   sub_dict=sub_dict)
-        ld.update([os.path.join(paths.UPDATES_DIR, '40-vault.update')])
+        self._ldap_update(['40-vault.update'])
```

Comment:
Was using DM here a relic of the past? It looks like dm_password in the class isn't used at all any more and it assumes ldapi=True because that's the URI that is generated. Would it make sense to clean that up as well?

Reply:
I came to the same conclusion. The argument is passed to the class and assigned to an instance var but the var is never used. I can remove the code.